routers/api/v1/repo/issue.go

```diff
@@ -218,7 +218,7 @@ func CreateIssue(ctx *context.APIContext, form api.CreateIssueOption) {
 		form.Labels = make([]int64, 0)
 	}
 
-	if err := issue_service.NewIssue(ctx.Repo.Repository, issue, form.Labels, assigneeIDs, nil); err != nil {
+	if err := issue_service.NewIssue(ctx.Repo.Repository, issue, form.Labels, nil); err != nil {
```

Comment:
Assignements are handled _after_ the issue is committed, alright. But assignee validation should occur before that. Otherwise someone using the API will realize too late that they made a mistake and we don't yet have a "delete issue" option. If I understood your code correctly, you've removed that validation from `models.newIssue()` (which makes sense because it will no longer create assignees), but I think they must be validated beforehand nonetheless. This should apply to all the other calls to `models.NewIssue()`

Reply:
> Surprisingly, if the assignee doesn't have right access the issue is never created, because that is already checked in "ValidateRepoMetas".

And NOW I've seen your comment. 😅